src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/Breadcrumb.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.ui.stepsdesign;
+
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class Breadcrumb {
+    ChromeDriver driver;
+    @Given("^I navigate to the home page$")
+    public void navigateToTheHomePage() throws InterruptedException {
```

Comment:
You're not using ElementHelper or Helper. Please use those methods.

You're not asserting anything in these tests either.

Too many `Thread.sleep` as well

You should be selecting by testid for a lot of these.

Please look at the other files in this directory for examples on how the tests should look.

Reply:
Done